src/mame/includes/fmtowns.h

```diff
@@ -372,8 +372,8 @@ class towns_state : public driver_device
 	void towns_crtc_refresh_mode();
 	void towns_update_kanji_offset();
 	void towns_update_palette();
-	void render_sprite_4(uint32_t poffset, uint32_t coffset, uint16_t x, uint16_t y, uint16_t xflip, uint16_t yflip, const rectangle* rect);
-	void render_sprite_16(uint32_t poffset, uint16_t x, uint16_t y, uint16_t xflip, uint16_t yflip, const rectangle* rect);
+	void render_sprite_4(uint32_t poffset, uint32_t coffset, uint16_t x, uint16_t y, uint16_t xflip, uint16_t yflip, uint16_t xhalfsize, uint16_t yhalfsize, const rectangle* rect);
+	void render_sprite_16(uint32_t poffset, uint16_t x, uint16_t y, uint16_t xflip, uint16_t yflip, uint16_t xhalfsize, uint16_t yhalfsize, const rectangle* rect);
```

Comment:
Since `xhalfsize` and `yhalfsize` are just tested for zero/nonzero, shouldn't they use `bool`?  As far as I can tell, `xflip` and `yflip` should be `bool` as well.

Reply:
Yeah, you're right, I was just copying Barry's code there but it makes more sense to pass the attributes as boolean. Changed.